core/block_validator.go

```diff
@@ -113,70 +111,25 @@ func (v *BlockValidator) ValidateState(block, parent *types.Block, statedb *stat
 	return nil
 }
 
-// VerifyHeaderWithSignature verifies the header with corresponding commit sigs
-func VerifyHeaderWithSignature(header *block.Header, commitSig []byte, commitBitmap []byte) error {
```

Comment:
The logic of VerifyHeaderWithSignature is similar to VerifySeal. But the cross shard receipts verification requires signatures and bitmap from different source. If you remove it, it will break cross shard tx.